code/modules/hydroponics/growninedible.dm

```diff
@@ -68,7 +68,7 @@
 
 /obj/item/weapon/grown/log/attackby(obj/item/weapon/W, mob/user, params)
 	..()
-	if(istype(W, /obj/item/weapon/circular_saw) || istype(W, /obj/item/weapon/hatchet) || (istype(W, /obj/item/weapon/twohanded/fireaxe) && W:wielded) || istype(W, /obj/item/weapon/melee/energy))
+	if(istype(W, /obj/item/weapon/circular_saw) || istype(W, /obj/item/weapon/hatchet) || (istype(W, /obj/item/weapon/twohanded/fireaxe) && W:wielded) || istype(W, /obj/item/weapon/melee/energy) || istype(W, /obj/item/weapon/twohanded/required/chainsaw))

```

Comment:
That wouldn't quite work since none of these have a sharpness of 2 apart from the esword. Besides, scalpels share that value as well, and it would be odd to chop wood with those. It would probably make the most sense to create a new sharpness bitflag for that, `IS_SHARP_BULKY` or something of the sort.